features/websocket.yml

```
@@ -1,7 +1,8 @@
-draft_date: 2024-07-11
-name: WebSockets Standard
-description: TODO
+name: WebSockets
+description: WebSockets provides a way to open a two-way communication channel between the user's browser and a server.
 spec: https://websockets.spec.whatwg.org/
+status:
+  compute_from: api.WebSocket
```

Let's use the entrypoint, the thing you must use in order to use the feature at all:

```suggestion
  compute_from: api.WebSocket.WebSocket
```

It doesn't make any difference in this case, but for some features it does, most notably when the constructor has been prefixed.